### PR TITLE
Path to screenshots changed for action takeScreenshot with custom path; screenshot warning added

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @AlexanderMoskovkin @inikulin @VasilyStrelyaev

### CODE REVIEW

File: test/functional/fixtures/api/es-next/take-screenshot/test.js [hunk 1]
```diff
@@ -17,19 +22,28 @@ if (!config.isTravisTask) {
...
-                    return checkScreenshotsCreated(true);
-                })
+                .then(checkScreenshotsAreNotCreated)
+                .then(function (success) {
+                    expect(testReport.screenshotPath).contains(SCREENSHOT_PATH_DO_NOT_SETTED_MESSAGE);
+                    expect(success).eql(true);
+                });
+        });
+
+        it('Should not take a screenshot without setting screenshotPath even if custom path spesified', function () {
```
Comment (VasilyStrelyaev) [inline test/functional/fixtures/api/es-next/take-screenshot/test.js]:
```
        it('Should not take a screenshot if screenshotPath is not set even if a custom path is specified', function () {
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
FPR

File: test/functional/fixtures/api/es-next/take-screenshot/test.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
-var config                  = require('../../../../config.js');
-var checkScreenshotsCreated = require('../../../../assertion-helper.js').checkScreenshotsCreated;
+var expect                        = require('chai').expect;
+var config                        = require('../../../../config.js');
+var checkScreenshotsCreated       = require('../../../../assertion-helper.js').checkScreenshotsCreated;
+var checkScreenshotsAreNotCreated = require('../../../../assertion-helper.js').checkScreenshotsAreNotCreated;
 
 
+var SCREENSHOT_PATH_MESSAGE_TEXT          = '___test-screenshots___';
+var SCREENSHOT_PATH_DO_NOT_SETTED_MESSAGE = '[cannot take screenshots because the screenshot directory is not specified]';
```
Comment (VasilyStrelyaev) [inline test/functional/fixtures/api/es-next/take-screenshot/test.js]: `SETTED`

Comment (VasilyStrelyaev) [inline test/functional/fixtures/api/es-next/take-screenshot/test.js]: `SCREENSHOT_PATH_NOT_SET_MESSAGE`
or - don't we use DIR now?
`SCREENSHOT_DIR_NOT_SET_MESSAGE`

File: src/runner/screenshots/screenshot-messages.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export default {
+    canNotBeCreated:     '[was unable to take a screenshot due to some error]',
```
Comment (VasilyStrelyaev) [inline src/runner/screenshots/screenshot-messages.js]: `canNot` - that's one word
`cannotBeCreated`

Comment (VasilyStrelyaev) [inline src/runner/screenshots/screenshot-messages.js]: why not `cannotCreate` BTW ?

**Comment by VasilyStrelyaev** [comment]
\r-

File: test/functional/assertion-helper.js [hunk 4]
```diff
@@ -66,38 +71,49 @@ exports.errorInEachBrowserNotContains = function errorInEachBrowserNotContains (
     }
 };
 
-exports.checkScreenshotsCreated = function checkScreenshotsCreated (withoutScreenshot, count, customPath) {
-    var screenshotsPath     = '___test-screenshots___';
-    var expectedCount       = count || 2;
-    var screenshotDirExists = isDirExists(screenshotsPath);
+exports.checkScreenshotsAreNotCreated = function () {
```
Comment (AlexanderMoskovkin) [inline test/functional/assertion-helper.js]: I don't expect any actions (like dir removing) from the this function by its name. Moreover we have the `checkScreenshotsCreated` function. I expect just
```js
checkScreenshotsAreNotCreated  () {
    return !checkScreenshotsCreated();
}
```

So, I suppose we should rename this. Maybe just `clearScreenshotsDir`?

Comment (AlexanderMoskovkin) [inline test/functional/assertion-helper.js]: Hm, just saw this. It's two different functions I suppose. 
The first checks screenshots. The second clears screenshots dir. And I think we should call the second after each test

Comment (AlexanderMoskovkin) [inline test/functional/assertion-helper.js]: lgtm for other

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by helen-dikareva** [comment]
FPR

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
just an unstable qunit, so we can review this

**Comment by AlexanderMoskovkin** [comment]
lgtm

File: src/test-run/browser-manipulation-manager.js [hunk 5]
```diff
@@ -36,7 +39,7 @@ export default class BrowserManipulationManager {
         catch (e) {
             // NOTE: swallow the error silently if we can't take screenshots for some
             // reason (e.g. we don't have permissions to write a screenshot file).
-            return SCREENSHOT_CAN_NOT_BE_CREATED_MESSAGE;
+            return SCREENSHOTS_WARNING_MESSAGES.cannotCreate;
```
Comment (inikulin) [inline src/test-run/browser-manipulation-manager.js]: Comment doesn't make sense now

File: src/test-run/browser-manipulation-manager.js [hunk 6]
```diff
@@ -1,10 +1,10 @@
 import uuid from 'uuid';
 import { resize as resizeWindow } from 'testcafe-browser-natives';
+import SCREENSHOTS_WARNING_MESSAGES from '../runner/screenshots/screenshot-messages';
```
Comment (inikulin) [inline src/test-run/browser-manipulation-manager.js]: it's better be named `warning-messages.js`

**Comment by inikulin** [comment]
\r-

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
lgtm

**Pull request merged**